app/assets/stylesheets/components/_modal.scss

```diff
@@ -18,24 +18,32 @@
   top: 0;
 }
 
+.modal-center {
+  display: table;
```

Comment:
I am still learning bass css, but I think you could add the `table ` class to achieve this style and `.col-12` to get 100% width. not sure if that is correct usage or how you'd get height. paging @hursey013